jjb/jobs/qcs-nightly-automation.yaml

```diff
@@ -41,17 +41,35 @@
                 sudo cp docker.conf /etc/sysconfig/docker
                 sudo systemctl start docker
                 sudo docker pull "${DOCKER_REGISTRY}/quipucords/quipucords:latest"
-                sudo docker run -d -p "443:443" -v /tmp:/tmp -v /home/jenkins/.ssh:/home/jenkins/.ssh -i "${DOCKER_REGISTRY}/quipucords/quipucords:latest"
+                # make log dir to save server logs
+                mkdir ${WORKSPACE}/log
+                # run container binding local ssh keys and log dir to the container
+                sudo docker run -d -p "443:443" -v /tmp:/tmp -v /home/jenkins/.ssh:/home/jenkins/.ssh -v ${WORKSPACE}/log:/var/log -i "${DOCKER_REGISTRY}/quipucords/quipucords:latest"
```

Comment:
Similar comment to the above:
Did you mean for `/tmp` to map to `/var/data`?
Did you mean for `/home/jenkins/.ssh` to map to `/sshkeys`?

Reply:
same reply